dotcom-rendering/docs/youtube/youtube-variations-2021-11-17.md

```diff
@@ -0,0 +1,26 @@
+ | | | MediaAtomBlockElement?| | | |?|
+ | | | MainMediaEmbedBlockComponent?| | | |?|
+DCR - AMP|First party| |[YoutubeBlockComponent](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/amp/components/elements/YoutubeBlockComponent.tsx)| |Y| |youtube-nocookie.com|[link](https://amp.theguardian.com/world/2021/jun/24/hong-kong-apple-daily-queue-final-edition-newspaper)
+| |Third party| |[VideoYoutubeBlockComponent](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/amp/components/elements/VideoYoutubeBlockComponent.tsx)| |Y| |youtube-nocookie.com|[link](https://amp.theguardian.com/music/musicblog/2015/may/27/stone-roses-spike-island-the-reality)
+Frontend|First party| |[youtube.scala.html](https://github.com/guardian/frontend/blob/main/common/app/views/fragments/atoms/youtube.scala.html), [youtube-player.ts](https://github.com/guardian/frontend/blob/main/static/src/javascripts/projects/common/modules/atoms/youtube-player.ts)|Partial|Y| |youtube.com / youtube-nocookie.com ***|[link](https://www.theguardian.com/world/2021/jun/24/hong-kong-apple-daily-queue-final-edition-newspaper?dcr=false)
+ | |Third party| | [VideoEmbedCleaner.scala](https://github.com/guardian/frontend/blob/main/common/app/views/support/cleaner/VideoEmbedCleaner.scala)|Y| |Y|youtube-nocookie.com|[link](https://www.theguardian.com/music/musicblog/2015/may/27/stone-roses-spike-island-the-reality?dcr=false)
+ | |Third party?| Commercial Hosted - Atoms | [youtube-player.ts](https://github.com/guardian/frontend/blob/main/static/src/javascripts/projects/common/modules/atoms/youtube-player.ts)|Partial|Y| |youtube.com|[link](https://www.theguardian.com/advertiser-content/microsoft-ai-for-earth/microsoft-ai-research)
+ | |Third Party | Commercial Hosted - Embeds | [youtube-player.ts](https://github.com/guardian/frontend/blob/main/static/src/javascripts/projects/common/modules/atoms/youtube-player.ts)| | | |youtube-nocookie.com|[link](https://www.theguardian.com/advertiser-content/radioactive-amazon-original-movie/radioactive)
+Editions (via Frontend) |First party| |[youtube-player.ts](https://github.com/guardian/frontend/blob/main/static/src/javascripts/projects/common/modules/atoms/youtube-player.ts)|Partial|Y| |youtube-nocookie.com|[link](https://embed.theguardian.com/embed/atom/media/d59557c3-d588-46bc-b8a6-cd7b626cd1a6#noadsaf)
+
+\* Previously the term 'embed' was used to refer to third party content and 'atom' for first party content. This has been updated to first and third party respectively to avoid confusion with other embeds.
+
+** Third party YouTube videos that originate from Composer embeds _should_ use the domain `youtube-nocookie`. However for freeform content (e.g. interactives, witness) there is the possibility that this is not adhered to.
+
+*** `youtube-player.ts` will default to the domain `youtube-nocookie.com` if consent is not given or an adfree setting is detected.
```

Comment:
If you want to accept the above suggestions…

```suggestion
<sup>1</sup> Previously the term 'embed' was used to refer to third party content and 'atom' for first party content. This has been updated to first and third party respectively to avoid confusion with other embeds.

<sup>2</sup> Third party YouTube videos that originate from Composer embeds _should_ use the domain `youtube-nocookie`. However for freeform content (e.g. interactives, witness) there is the possibility that this is not adhered to.

<sup>3</sup> `youtube-player.ts` will default to the domain `youtube-nocookie.com` if consent is not given or an adfree setting is detected.
```